docs/components/Mdx/Props.js

```diff
@@ -120,11 +120,9 @@ export function Props({ propTypes }) {
                     </Table.Td>
                     <Table.Td textAlign="center" w="15%">
                       {required && (
-                        <Tooltip content="is required">
```

Comment:
why did you removed tooltip ?

Reply:
we've got a bug in this component i think, try to add a div between tooltip and tag

Reply:
no improvement there : 

> Warning: An error occurred during hydration. The server HTML was replaced with client content in "div"`. 

> See more info here: https://nextjs.org/docs/messages/react-hydration-error

> Warning: Received 'true' for a non-boolean attribute 'modal'.
If you want to write it to the DOM, pass a string instead: modal="true" or modal={value.toString()}.`

Is this a problem related to Reakit ?

Reply:
Hey @Geminii I think I've already had this issue when the first child of `tooltip`is a div. Could you please try `<Tooltip content="is required" fixed>` to avoid this error?

Reply:
@cnairiwttj We continued to test and investigate with @theo-mesnil but no result for the moment. I tried to add `fixed` like you suggest but no improvement 😭

Reply:
This information is useless on the ux we removed it for now :)